docs/contributors_guide/maintenance/iso_release_procedures/Pre_Release_Checklist.rst

```diff
@@ -439,41 +431,7 @@ tested in acceptance/simp-packer tests) do function as advertised:
   - ``gen_ldap_update``
   - ``updaterepos``
 
-* The HOWTO documentation is still correct.
-
-  - HOWTO Back up the Puppet Master
-  - HOWTO Change Puppet Servers
-  - HOWTO Configure a Puppet Server Behind a NAT
-  - HOWTO Configure iptables NAT Rules
-  - HOWTO Configure NFS
-  - HOWTO Configure SNMPD
-  - HOWTO Disable SSH
-  - HOWTO Discard Mail to Root
-  - HOWTO Enable Kerberos
-  - HOWTO Enable Redundant LDAP
-  - HOWTO Enable SFTP Restricted Accounts
-  - HOWTO Exclude YUM Repositories
-  - HOWTO Manage Workstation Infrastructures
-  - HOWTO Modify the Nightly Update Schedule
-  - HOWTO Modify The Puppet Cron Schedule
-  - HOWTO Restrict Network Access to SSH
-  - HOWTO Setup a SIMP Control Repository
-  - HOWTO Set up Central Log Collection
-  - HOWTO Setup SSH Authorized Keys
-  - HOWTO Upgrade SIMP
-  - HOWTO Work with the SIMP Rsync Shares
-
-Verify SIMP server is compliant with security standards
```

Comment:
@trevor-vaughan This seems like an important step that should be done.

Reply:
@lnemsick-simp It is an important step but we don't need to list all of them. I just linked over to all of the HOWTO guides below.